main/settings.py

```diff
@@ -219,6 +219,13 @@
     required=True,
 )
 
+FEATURE_ENABLE_CERTIFICATES_FOR_USERS = get_bool(
```

Comment:
Could we use `FEATURE_ENABLE_CERTIFICATES` as these certificates are only generated for the enrolled users so, in my opinion, no need to mention users?

Reply:
This name sounds good, but my intention of using `FOR_USERS` was that the whole certificate feature doesn't get blocked by this. Admin can still create certificates and certificate pages. Only the user's view will be affected. Does this change your mind?

Reply:
In that case, I would go with the @pdpinch suggestion `FEATURE_ENABLE_CERTIFICATE_DASHBAORD_LINKS `.

Reply:
What about `ENABLE_CERTIFICATE_USER_VIEW`?